app/config/security.yml

```diff
@@ -9,19 +11,24 @@ security:
     providers:
         in_memory:
             memory:
-                users: %security_users%
+                # Only for demo purposes. Of course, use a more
+                # secure password on production.
+                users:
+                    anna_admin: { password: kitten, roles: ROLE_ADMIN }
```

Comment:
i think we intended the user account to demo the publish workflow. but if you checked and we don't do anything with it we either never did it or it got removed again.